core/src/main/scala/kafka/tools/ConsumerPerformance.scala

```diff
@@ -201,8 +207,22 @@ object ConsumerPerformance {
       .withRequiredArg
       .describedAs("count")
       .ofType(classOf[java.lang.Integer])
-      .defaultsTo(1)
+      .defaultsTo(1)    
+    val propsFileOpt = parser.accepts("consumer.config", "External consumer properties file.")
+      .withRequiredArg
+      .describedAs("consumer properties")
+      .ofType(classOf[java.lang.String])
+      .defaultsTo("")
+    val freeformPropsOpt = parser.accepts("consumer-properties", "Freeform consumer properties in the format key=val,key=val,key=val...")
+      .withRequiredArg
+      .describedAs("consumer properties")
+      .ofType(classOf[java.lang.String])
```

Comment:
Just noticed that `ConsoleProducer` has the following:

```
    val producerPropertyOpt = parser.accepts("producer-property", "A mechanism to pass user-defined properties in the form key=value to the producer. ")
            .withRequiredArg
            .describedAs("producer_prop")
            .ofType(classOf[String])
```

And it uses `CommandLineUtils.parseKeyValueArgs(options.valuesOf(propertyOpt))`. We probably should be using the same naming and parsing code here.